### ISSUE TITLE
Domoticz

### ISSUE THREAD

**Issue by Apollon77** [ioBroker/AdapterRequests#41]
domoticz ist in C/C++ geschrieben und nativ verfügbar für Raspberry Pis, Windows, Linux, OS X und für einige NAS-Syteme. Als Scripting-Engine wird Lua verwendet, über die mit Blockly die Automatisierung vorgenommen wird. Leider Homematic nicht unterstützt, jedoch ist die Liste an unterstützter Hardware recht umfangreich.

https://github.com/domoticz/domoticz
https://github.com/Wifsimster/domoticz-api (NodeJS API)

https://github.com/patchworkboy/homebridge-eDomoticz

Hier noch ein Link als Info für den Entwickler - https://www.domoticz.com/wiki/Domoticz_API/JSON_URL's

http://forum.iobroker.net/viewtopic.php?f=20&t=6575&p=68658#p68658

Integration erschwert da kein Event-Bus vorhanden ist, API kan gelesen werden und gibt objecten als JSON tabelle wieder.

Ich habe scripting erstellt um:

Die objecte zu lesen und im iObroker erstellen
Per MQTT nach updates von domitcz lauschen und damit werte updaten
Resultat: Syncronisation von objecten moeglisch und empfangen von aenderungen.
Bis jetzt nicht weiter gekommen als buttons und switches da mir andere gerate fehlen um zu testen.
Code selber ist auch kinderschuhe

Werde den code ein biscchen aufraumen und als beispiel im Forum Posten.

Migrated from Trello with 0 votes

**Comment by Apollon77** [comment]
Homebridge modules exist, see above

Please check if the new ioBroker.ham Adapter can help here: see https://forum.iobroker.net/viewtopic.php?f=36&t=14972

**Comment by salopette** [comment]
Hello, how can this install in the IoBroker?

**Comment by DutchmanNL** [comment]
> Hello, how can this install in the IoBroker?

what do you mean ?

this is an request to integrate with domoticz, the HomeBridge plugin works see HAM-adapter

**Comment by salopette** [comment]
I thought you can use it to access and control domoticz from the IoBroker?
Or have I misunderstood that?

**Comment by DutchmanNL** [comment]
> I thought you can use it to access and control domoticz from the IoBroker?
> Or have I misunderstood that?

yes you can, as mentioned we do not have an native integration but you can use the HAM (Homebridge) adapter with the domoticz plugin : 

HAM adapter : https://github.com/ioBroker/ioBroker.ham
Domoticz Plugin : https://github.com/patchworkboy/homebridge-eDomoticz#readme

if there is a large needed of interest I can work on an adapter but just for us 2 that's a little overkill.
Are you maybe from NL ? (domoticz is used a lot there as I know)

If we have more people interest or funding maybe a dedicated adapter can be build

**Comment by salopette** [comment]
No, I am from DE,
I have been using Domoticz for some time and now I have discovered IoBroker and wanted to try it out and operate both systems in parallel.

My English and programming skills are = 0

**Comment by BassT23** [comment]
Hi @ll

Zuhause benutze ich ohne www Weiterleitung iobroker.
Auf meiner Baustelle nutze ich wg wenig Ressourcen domoticz. Dieses System ist online.

Würde nun gerne Domoticz in mein ioBroker "integrieren".

Werde es mal mit Ham und plugin versuchen.

CU

**Comment by DutchmanNL** [comment]
> Würde nun gerne Domoticz in mein ioBroker "integrieren".
> 
> Werde es mal mit Ham und plugin versuchen.

bin gespannt auf deine Erfahrungen.
Wie vorher erwähnt ist momentan die potentielle user zahl 2 a 3, en mehr bedarf lohnt es sich a einem adapter zu arbeiten
